src/SuperSocket.MCP/McpHttpServer.cs

```diff
@@ -123,11 +122,15 @@ public async Task HandleHttpRequestAsync(McpHttpRequest httpRequest, IAppSession
             else if (path == "/mcp/capabilities" && httpRequest.HttpRequest.Method == "GET")
             {
                 // Handle capability inquiry
+                var toolHandlers = _handlerRegistry.GetToolHandlers();
+                var resourceHandlers = _handlerRegistry.GetResourceHandlers();
+                var promptHandlers = _handlerRegistry.GetPromptHandlers();
+                
                 var capabilities = new McpServerCapabilities
                 {
-                    Tools = new McpToolsCapabilities { ListChanged = true },
-                    Resources = new McpResourcesCapabilities { Subscribe = true, ListChanged = true },
-                    Prompts = new McpPromptsCapabilities { ListChanged = true },
+                    Tools = toolHandlers.Any() ? new McpToolsCapabilities { ListChanged = true } : null,
+                    Resources = resourceHandlers.Any() ? new McpResourcesCapabilities { Subscribe = true, ListChanged = true } : null,
+                    Prompts = promptHandlers.Any() ? new McpPromptsCapabilities { ListChanged = true } : null,
                     Logging = new McpLoggingCapabilities()
                 };
```

Comment:
Why do we still need to do swtich/case here? I thought we can route logic operations to commands created in this PR by path?

Reply:
if/esle (switch/case) for path is still there, can we just reuse SuperSocket commands?